functions/Get-DbaAgentSchedule.ps1

```diff
@@ -0,0 +1,78 @@
+FUNCTION Get-DbaAgentSchedule
```

Comment:
thanks @PowerDBAKlaas - please remember to rename the file as well, @DevOpsFu. @sanderstad  is not working on Gets so this is perfect 👍